junit-platform-engine/src/main/java/org/junit/platform/engine/discovery/ClasspathResourceSelector.java

```diff
@@ -59,6 +65,14 @@ public String getClasspathResourceName() {
 		return this.classpathResourceName;
 	}
 
+	/**
+	 * Get the selected position within the classpath resource as a
+	 * {@link FilePosition}.
+	 */
+	public FilePosition getPosition() {
```

Comment:
JUnit 5 seems to follow the pattern that nullable return values are always wrapped in an Optional.

https://github.com/junit-team/junit5/blob/3f7fed61f2edcee2a5551dbde75382b1dbe21267/junit-platform-console/src/main/java/org/junit/platform/console/tasks/TreeNode.java#L74-L76